### PR TITLE
refactor: remove obsolete code

### PR DESCRIPTION
**Pull request by matangeorgi**
- Unify some of the css into a general lessons stylesheet
- Delete obsolete stylesheets
- Delete comments

### CODE REVIEW

**Comment by matangeorgi** [comment]
@ShireeWill please take a look to validate that no regression were presented here, its only code refactor.

File: app/routes/01-lets-start/lesson-01.module.scss [hunk 1]
```diff
@@ -1,30 +1,11 @@
...
-    height: $playground-height;
-
-    // lesson specific
-    background-image: url('../../../src/assets/lesson-01-bg.png');
-    background-size: cover;
-    position: relative;
-  }
+.playground {
+  background-image: url('../../../src/assets/lesson-01-bg.png');
+  background-size: cover;
```
Comment (DanielGovrin) [inline app/routes/01-lets-start/lesson-01.module.scss:6]: You removed the .root class, so the <div> that used to apply this class is no longer relevant.

File: app/routes/02-elements-panel/lesson-02.module.scss [hunk 2]
```diff
@@ -1,34 +1,12 @@
...
-    display: grid;
-    grid-template-columns: 250px;
-    grid-template-rows: 60px 60px 60px 60px;
-    padding: 130px 75px 130px 75px;
-    gap: 20px;
-  }
+.playground {
+  display: grid;
+  padding: 130px 75px 130px 75px;
+  gap: 20px;
```
Comment (DanielGovrin) [inline app/routes/02-elements-panel/lesson-02.module.scss:7]: The comment regarding the removal of the root style applies to this lesson as well as all other lessons components.

File: app/routes/05-styles-panel/lesson-05.module.scss [hunk 3]
```diff
@@ -1,31 +1,13 @@
 @use '../../../src/globals/variables.module.scss' as *;
+@use '../../../src//globals/lessons.module.scss' as *;
 
-.root {
+.playground {
```
Comment (DanielGovrin) [inline app/routes/05-styles-panel/lesson-05.module.scss:4]: This styling dont do anything

**Review by DanielGovrin — APPROVED** [review]

**Pull request merged**